docs: fix broken link to themes documentation (#49228)
keycloak/keycloak: docs/documentation/server_admin/topics/realms/themes.adoc

  Whenever {project_name} has to send out an email, it uses templates defined in this theme to craft the email.

.Additional resources

> * The link:{developerguide_deploying_themes}[UI Customization Guide] describes how to create a new theme or modify existing ones.

Reviewer: ```suggestion
* For more details about how to create or modify a theme , see ​{ui_customization_guide_themes}[Working with themes] .
* 

```
